breathe/renderer/sphinxrenderer.py

```diff
@@ -523,8 +523,6 @@ def debug_print_node(n):
                 names.extend(node.compoundname.split('::'))
                 break
 
-        names.reverse()
```

Comment:
This is needed as the code walks up the stack of XML nodes while appending items.